Makefile

```diff
@@ -249,3 +249,12 @@ catalog-push: ## Push a catalog image.
 .PHONY: e2e-test
 e2e-test: ## Run e2e tests for the controller
 	go test ./tests/e2e/ -run ^TestOdhOperator -v --kf-namespace=${OPERATOR_NAMESPACE} ${E2E_TEST_FLAGS}
+
+.PHONY: get-tar
+get-tar: ## Get latest odh-manifests tarball
+	curl -s https://api.github.com/repos/opendatahub-io/odh-manifests/releases/latest \
+	| grep "tarball_url" \
+	| cut -d ":" -f2,3 \
+	| sed 's/,/ /g' \
+	| tr -d \" \
+	| wget -qi - -O odh-manifests.tar.gz 
```

Comment:
We can simplify this to run `wget` on the github tarball url for the `master` in the same format that we use for existing kfdef objects so a simple 

```bash
wget ${MANIFESTS_TARBALL_URL} -O odh-manifests.tgz
```

Can you use a `Makefile` variable for specifying the tarball url to dowload from?